Jenkinsfile

```diff
@@ -1,6 +1,6 @@
 pipeline {
     agent {
-        label 'builder-backend-j8 || new-builder-backend-j8'
```

Comment:
what was special about this builder?

Reply:
Nothing, it was a general purpose builder. Certain functions needed dedicated builders (like the agent), so we started adding builder labels to all jobs